website/src/documentation/transforms/python/element-wise/withtimestamps.md

```diff
@@ -19,10 +19,116 @@ limitations under the License.
 -->
 
 # WithTimestamps
+
+<script type="text/javascript">
+localStorage.setItem('language', 'language-py')
+</script>
+
 Assigns timestamps to all the elements of a collection.
 
 ## Examples
-See [BEAM-7389](https://issues.apache.org/jira/browse/BEAM-7389) for updates. 
 
-## Related transforms 
-* [Reify]({{ site.baseurl }}/documentation/transforms/python/elementwise/reify) converts between explicit and implicit forms of Beam values.
\ No newline at end of file
+In the following examples, we create a pipeline with a `PCollection` and attach a timestamp value to each of its elements.
+Timestamps are especially useful on streaming pipelines where windowing and late data play a more important role.
+
+### Example 1: Timestamp by event time
+
+Often times, the elements themselves already contain a timestamp field that can be used.
+`beam.window.TimestampedValue` will take a value and a timestamp in the form of seconds, where a unix timestamp can be used.
```

Comment:
"`beam.window.TimestampedValue` takes a value and a Unix timestamp in the form of seconds."

^is this sentence still accurate? Else try shortening the sentence:

"`beam.window.TimestampedValue` takes a value and a timestamp in the form of seconds."

and you have a comment after this section on converting to Unix timestamps.

Reply:
Thanks, it's still accurate so changing it. I left the Unix part to make it explicit.